src/page-modules/departures/server/journey-planner/index.ts

```diff
@@ -34,17 +34,12 @@ import {
   ServiceJourneyData,
   serviceJourneySchema,
 } from './validators';
+import { TransportMode as GraphQlTransportMode } from '@atb/modules/graphql-types';
```

Comment:
@mikaelbr, what do you think about the `TransportMode` in `modules/graphql-types`? Should that one be left alone and used to cast like it's done here?

Reply:
Haven't seen every detail here, but at least generally I think the type from graphql should be converted into something "safer" at first interaction with the type in some way. So inside API fn mapping typically

Reply:
Looks like it's handled in some way. When mapping from GraphQL (https://github.com/AtB-AS/planner-web/blob/ede4246d07b409a33ab06e83afc0ad2b7a331309/src/page-modules/departures/server/journey-planner/index.ts#L119) we check the transport mode that is received and parse it with zod against the schema (https://github.com/AtB-AS/planner-web/blob/ede4246d07b409a33ab06e83afc0ad2b7a331309/src/modules/transport-mode/utils.ts#L25-L43). When passing to GraphQL it is currently just casted to the GraphQL type since `'bus' === 'bus'`.